src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -51,6 +51,7 @@ data class Referral(
   var desiredOutcomesIDs: List<UUID>? = null,
   var completionDeadline: LocalDate? = null,
 
+  @NotNull @ManyToOne(fetch = FetchType.LAZY) val intervention: Intervention,
```

Comment:
The @NotNull means the intervention can't be null - but doesn't a referral get created without assigning an intervention to it?